client/src/pages/proposalDetails/ProposalButton.js

```diff
@@ -0,0 +1,21 @@
+import './ProposalButton.css';
+
+function ProposalButton({buttonText, isVotingButton, buttonClassName, onClickFunc}) {
+  if (isVotingButton) {
+    return (
+      <div>
+        <button className={buttonClassName} onClick={onClickFunc}>
+          {buttonText}
+        </button>
+      </div>
+    );
+  } else {
+    return (
+      <button className="genericProposalButton" onClick={onClickFunc}>
+          {buttonText}
+      </button>
+    );
+  }
+}
```

Comment:
why is buttonClassName only used if isVotingButton is true? If you want general button behavior aand then a specialized case for voting button, consider writing 2 different button functions